src/common/Types.ts

```diff
@@ -585,9 +621,15 @@ export interface ViewPropsShared extends CommonProps, CommonAccessibilityProps {
     shouldRasterizeIOS?: boolean; // iOS-only prop, if view should be rendered as a bitmap before compositing
     viewLayerTypeAndroid?: ViewLayerType; // Android only property
 
-    restrictFocusWithin?: boolean; // Web-only, during the keyboard navigation, the focus will not go outside this view
+    restrictFocusWithin?: RestrictFocusType; // Web-only, during the keyboard navigation, the focus will not go outside this view
```

Comment:
This appears to be a  breaking change. We'll need to leave this as a boolean value and find another way of exposing this new functionality if it's needed.